src/main/java/com/thealgorithms/datastructures/hashmap/hashing/ItineraryTickets.java

```diff
@@ -0,0 +1,44 @@
+package com.thealgorithms.datastructures.hashmap.hashing;
+
+import java.util.HashMap;
+
+/**
+ * This class represents an itinerary of tickets. Given a list of tickets, it finds the
+ * complete itinerary.
+ */
+public class ItineraryTickets {
+
+    public static void main(String[] args) {
+        HashMap<String, String> tickets = new HashMap<>();
+        tickets.put("Chennai", "Bengaluru");
+        tickets.put("Mumbai", "Delhi");
+        tickets.put("Goa", "Chennai");
+        tickets.put("Delhi", "Goa");
+
+        String start = getStart(tickets);
+        System.out.print(start);
+        for (String key : tickets.keySet()) {
+            System.out.print(" -> " + tickets.get(start));
+            start = tickets.get(start);
+        }
+    }
+
+    /**
+     * This method finds the starting point of the itinerary.
+     *
+     * @param tickets A map of tickets where the key is the departure point, and the value is the destination.
+     * @return The starting point of the itinerary.
+     */
+    public static String getStart(HashMap<String, String> tickets) {
+        HashMap<String, String> revTickets = new HashMap<>();
+        for (String key : tickets.keySet()) {
```

Comment:
can you add tests?

Reply:
Hey sorry late reply.
by test we mean adding test cases?
If yes, Then can you guide me what would I have to actually do for it.
As I am very new here, but am eager to learn.
In any case please do reply.🙏🏻

Reply:
**Update**: I have added JUnit tests for the `ItineraryTickets` class to address the testing requirements. You can review the changes in the updated code.

If you have any questions or need further changes , please feel free to let me know. Thank you for your time and feedback!

Reply:
I am unable to resolve clang format linter.
I don't know what it is...
I don't know what to do now, @siriak can you please guide me what to do next.
I want my pull request to me merged (as this would be my first PR to be merged)
Please Help me out.